practices/optimizations.md

```diff
@@ -0,0 +1,55 @@
+---
+slug: optimizations
+stages:
+  - development
+short_description: Optimization is the process of modifying a software system to make some aspect of it work more efficiently or use fewer resources
+tags:
+  - optimizations
+  - code coverage
+keywords:
+  - code  - quality
+  - merge
+  - testing
+  - codebase
+---
+
+# Optimizations
+
+**TL;DR**
+
+Optimization is an act, process, or methodology of making something (such as a design, system, or decision) as fully perfect, functional, or effective as possible
```

Comment:
Please, use your own definition. Or quote the dictionary you have used: _an act, process, or methodology of making something (such as a design, system, or decision) as fully perfect, functional, or effective as possible_ from https://www.merriam-webster.com/dictionary/optimization. It is unethical to use someone's definition without credits. I would prefer your definition.

Reply:
Done